src/lib/zod/formHelpers.ts

```diff
@@ -60,11 +60,13 @@ export const zUsername = z
 
 export const zDescription = z.string().min(3).max(160);
 
-const urlRegex = /^https?:\/\/([\w\d-]+\.){1,}[\w\d-]+(\/[\w\d-]+)*\/?$/;
+const url = z
```

Comment:
?? is z.url the same?

Reply:
its bettter, it tolerates query strings and dots and stuff